packages/formstr-app/src/containers/CreateForm/index.style.ts

```diff
@@ -1,16 +1,16 @@
 import styled from "styled-components";
-import { MEDIA_QUERY_MOBILE_TABLET } from "./utils/css";
+import { MEDIA_QUERY_MOBILE } from "../../utils/css";
```

Comment:
?

Reply:
moved this file in global utils, as it's going to be used by multiple containers